### ISSUE TITLE
#a11y: Add ALT text to images 

### ISSUE THREAD

**Issue by hejjoe** [jfilter/ifg-ticker#23]
It should be possible to add an alt text to tweets, which are created via the twitter api. Ideally the alt text contains all the words of the text snippet in the uploaded image.
https://blog.twitter.com/developer/en_us/a/2016/alt-text-support-for-twitter-cards-and-the-rest-api
https://developer.twitter.com/en/docs/twitter-api/v1/media/upload-media/api-reference/post-media-metadata-create

**Comment by hejjoe** [comment]
It seems be to be an old issue with twython :/ https://github.com/ryanmcgrath/twython/issues/420
